### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by qseeker** [LucasBullen/corrosion#28]
Hi!
Thank you for cool project! I can't wait to play with it!

I got following error message then try to install the plugin. I tried to install it in Eclipse Neon3 CDT using p2 repository:

```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Corrosion: Rust edition in Eclipse IDE 0.1.0.201802061310 (org.eclipse.corrosion.feature.feature.group 0.1.0.201802061310)
  Missing requirement: Rust edition in Eclipse IDE 0.1.0.201802061310 (org.eclipse.corrosion 0.1.0.201802061310) requires 'bundle org.eclipse.ui.genericeditor 1.0.1' but it could not be found
  Cannot satisfy dependency:
    From: Corrosion: Rust edition in Eclipse IDE 0.1.0.201802061310 (org.eclipse.corrosion.feature.feature.group 0.1.0.201802061310)
    To: org.eclipse.corrosion [0.1.0.201802061310]
```

Eclipse Oxigen 2 CDT:
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: Corrosion: Rust edition in Eclipse IDE 0.1.0.201802061310 (org.eclipse.corrosion.feature.feature.group 0.1.0.201802061310)
  Missing requirement: Language Server Protocol client for Eclipse IDE (Incubation) 0.6.0.201801160955 (org.eclipse.lsp4e 0.6.0.201801160955) requires 'bundle org.eclipse.jface.text 3.13.0' but it could not be found
  Cannot satisfy dependency:
    From: Corrosion: Rust edition in Eclipse IDE 0.1.0.201802061310 (org.eclipse.corrosion.feature.feature.group 0.1.0.201802061310)
    To: org.eclipse.lsp4e [0.6.0.201801160955]

```

No mater via Marketplace or using p2 repository - the result is same.

**Comment by dhartford** [comment]
confirm similar issues on both (same day testing it out as well).  Not sure if the recent renaming caused something weird, but the Oxygen.2 with the jface dependency seems more about the language protocol integration.

**Comment by LucasBullen** [comment]
Thank you for catching and reporting this issue. The issue has been addressed with c2b13af and tested on Oxygen.2 and Photon. Please comment if otherwise unsuccessful. We will not be supporting any versions of Eclipse older than Oxygen at this time.

**Comment by dhartford** [comment]
looks good on install now, thanks for quick resolution!

corrosion: 0.1.0.201802080934
language protocol client: 0.5.0.201802011509
IDE: oxygen.2
